src/impact-dashboard/EpidemicModelContext.tsx

```diff
@@ -22,22 +22,40 @@ export enum RateOfSpread {
 // fields that we want to store
 export interface ModelInputsPopulationBrackets {
   age0Cases?: number;
+  age0Recovered?: number;
   age0Population?: number;
+  age0Deaths?: number;
```

Comment:
Slight order inconsistency here. Should move this line up one, and the line directly below one, to be consistent in the _cases, recovered, deaths, population_ ordering.